backend/corpora/common/corpora_orm.py

```diff
@@ -323,6 +323,7 @@ class DbDataset(Base, AuditMixin, TimestampMixin):
     development_stage = Column(JSONB)
     cell_type = Column(JSONB)
     cell_count = Column(Integer)
+    donor_id = Column(ARRAY(String), nullable=True)
```

Comment:
question: It makes sense to set nullable=True so we can do the migration with the existing corpus. Should we add a TODO/ticket to update this after the migration to remove nullable=True, since it's a required field?

Reply:
I think `nullable=True` is the default and I wanted to be explicit here. I don't think that enforcing non-nullability is important since this will already be handled by the pipeline.